ferrocene/tools/self-test/src/main.rs

```diff
@@ -42,20 +48,23 @@ fn main_inner(reporter: &dyn Reporter) -> Result<(), Error> {
     let mut targets = targets::check(reporter, &sysroot)?;
     linkers::check_and_add_rustflags(reporter, &environment, &sysroot, &mut targets)?;
     compile::check(reporter, &sysroot, &targets)?;
+    report_linker_flags(reporter, &targets);
+
+    reporter.success("Ferrocene self-check completed!");
+    Ok(())
+}
 
+fn report_linker_flags(reporter: &dyn Reporter, targets: &[Target]) {
```

Comment:
was this just to improve readability of main_inner

Reply:
Yes. I was thinking if we can move it into the `linkers::check_and_add_rustflags`? It feels like it should go in there.

Reply:
seems like a perfect place to do it